dipy/denoise/noise_estimate.py

```diff
@@ -103,7 +105,8 @@ def piesno(data, N, alpha=0.01, l=100, itermax=100, eps=1e-5, return_mask=False)
         q = 0.5
 
     # Initial estimation of sigma
-    initial_estimation = np.percentile(data, q * 100) / np.sqrt(2 * _inv_nchi_cdf(N, 1, q))
+    initial_estimation = np.percentile(data, q * 100) / \
```

Comment:
Same here, line was too long to not break up the math, how much character does it take in the old form?